From 8876073eda4fe8e133143a1effec56fa40bc5dab Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Fri, 2 Dec 2011 14:05:35 +0800
Subject: [PATCH] rk29: last_log: support /proc/last_kmsg when
 ANDROID_RAM_CONSOLE is not enabled, better support android bugreport

---
 arch/arm/mach-rk29/last_log.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-rk29/last_log.c b/arch/arm/mach-rk29/last_log.c
index 7096001eb377..6e6ead70372b 100644
--- a/arch/arm/mach-rk29/last_log.c
+++ b/arch/arm/mach-rk29/last_log.c
@@ -65,6 +65,10 @@ static int __init last_log_init(void)
 	entry->proc_fops = &last_log_file_ops;
 	entry->size = LOG_BUF_LEN;
 
+#ifndef CONFIG_ANDROID_RAM_CONSOLE
+	proc_symlink("last_kmsg", NULL, "last_log");
+#endif
+
 	return 0;
 }
 
-- 
2.35.3

